spotify/translations/nl.yaml

```diff
@@ -0,0 +1,25 @@
+---
+configuration:
+  log_level:
+    name: Log-niveau
+    description: >-
+      Bepaalt het niveau van logboekdetails die de add-on biedt.
+  name:
+    name: Naam
+    description: >-
+      De naam van je device (het Spotify Connectie-doel), zoals het er
+      uit ziet op de officiële Spotify-cliënten.
+  bitrate:
+    name: Bitsnelheid
+    description: >-
+      De bitrate die Spotify moet gebruiken. Hoe hoger, hoe beter de geluidskwaliteit,
+      de add-on verbruikt echter meer gegevens.
```

Comment:
'[..] gegevens.' reads strange in my opinion. Maybe call it 'internet'?

"[...] de add-on verbruikt echter meer internet."